fix: use new api to get file (bunkr)

NTFSvolume opened the pull request:
- Use the new API to get the file URL from get.bunkr.su
- Now 1 to 3 requests per item in an album are needed.
- Use the new scrape item syntax
- Remove functions and methods no longer in use
- Needs #712 for the new scrape item syntax

sublimal commented:
More elegant/pythonic decode:

```python
import binascii
import itertools
import math
def decode_encrypted_url(url, timestamp):
    arr = binascii.a2b_base64(url)
    secret = f"SECRET_KEY_{math.floor(timestamp/3600)}"
    return "".join(chr(c ^ ord(k)) for c, k in zip(arr, itertools.cycle(secret)))
 ```

NTFSvolume commented:
## Changes

- Removed `@error_handling_wrapper` from `handle_reinforced_link` because it should always be called from another method that already has it
- Added a new method `reinforced_file` to handle `get.bunk.su/file/<file_id>` as input
- Save the full reinforced URL as referer instead just `https://get.bunk.su/` (when available)
- Added some of the code from the suggestion of the decrypt function but still keeping it readable

NTFSvolume commented:
Also switched back to the current syntax for the album setup, to merge it without #712
